tests/lib/rules/sort-prop-types.js

```diff
@@ -2325,16 +2325,16 @@ ruleTester.run('sort-prop-types', rule, {
     {
       code: `
         const Foo = (props: {
-          zzz: string,
-          aaa: string,
+          zzz: string;
+          aaa: string;
```

Comment:
let's not change existing test cases - we need to be able to handle any mix of commas and semicolons, since TS types can have both